src/librustc/ty/context.rs

```diff
@@ -1408,6 +1408,7 @@ impl<'tcx> TyCtxt<'tcx> {
     }
 
     pub fn encode_metadata(self)-> EncodedMetadata {
+        let _prof_timer = self.prof.generic_activity(dbg!("generate_crate_metadata"));
```

Comment:
Uh, dbg! here needs to be removed :)

Reply:
Ah, good catch! That `dbg!` is very sneaky.

Reply:
Whoops, sorry!